automatic.crontab

```diff
@@ -62,7 +62,8 @@ poStatusHtml=/home/nvdal10n/ikiwiki/publish/poStatus.html
 02  4 * * fri $AddonTranslationUpdate debugHelper
 
 #x  5 for addons starting with e
-00  5 * * fri $AddonTranslationUpdate emoticons
+00  5 * * fri $AddonTranslationUpdate easyTableNavigator
```

Comment:
Please add `easyTableNavigator` at the of the 'e' section. At '20 5 * * fri $AddonTranslationUpdate easyTableNavigator`